.github/actions/asf-release-candidate/action.yml

```diff
@@ -0,0 +1,99 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
One of the reasons I've marked this as a draft is because I don't think this action wants to live in the daffodil repo. When publishing an action you either need to commit the `node_modules` directory (which is ~130MB), or essentially statically 'compile' the the main.js/post.js files and commit those. I'm not sure that would be appropriate in the Daffodil repo. Feels like we either need a separate daffodil-infrastructure repo, or maybe use daffodil-extra or something. The build-release container might also want to live there since it also isn't specifically about daffodil since it also works for vs code and sbt extensions.

Reply:
Another thought about where the action could live is in https://github.com/apache/infrastructure-actions--it is *almost* generic enough to be useful for other ASF projects and could probably be made useful with some tweaks. That said, I imagine ASF would want to see it in use before being merged, but if that's the right place we could start a discussion and see what they say. It would be nice if this could be something maintained by the ASF community rather than just us. I know I wish I had this around when we started, it would have avoided a lot of complexities.

Reply:
> Another thought about where the action could live is in https://github.com/apache/infrastructure-actions

This seems ideal.

Reply:
I've created a PR for infrastructure-actions with some minor tweaks, we'll see what the say.

Reply:
It's only been a few days, but there doesn't seem much interest by INFRA to include the action in infrastructure-actions. Infra is usually pretty busy so I imagine it's no high priority. So I suggest for now we keep the action in a `daffodil-*` repo, and if ASF ever does show interest we can can always migrate to that version. Also, maybe seeing it in action would make it a bit more appealing.

Any thoughts on where it should live in the meantime? Previously I proposed a new repo called daffodil-infrastructure--something that is not so user focused like daffodil-extra, but is more focused on things the various daffodil repos can share for releases, CI, etc. Thoughts?

Reply:
I thought I responded to this.... sorry. I think creating daffodil-infrastructure seems sensible. Daffodil extra I was enviioning as just a source repo for little stuff, along the lines of OpenDFDL but official for our project.